src/main/java/seedu/homerce/commons/core/Messages.java

```diff
@@ -16,6 +17,11 @@
     public static final String MESSAGE_INVALID_APPOINTMENT_DISPLAYED_INDEX = "The appointment"
             + " index provided is invalid";
     public static final String MESSAGE_APPOINTMENTS_LISTED_OVERVIEW = "%1$d appointments listed!";
+    public static final String MESSAGE_APPOINTMENT_ALREADY_DONE = "This appointment is already marked as done!";
+    public static final String MESSAGE_APPOINTMENT_ALREADY_UNDONE = "This appointment is already marked as not done!";
+    public static final String MESSAGE_DUPLICATE_APPOINTMENT = "This appointment already exists in the homerce book.";
```

Comment:
Maybe using "exists in Homerce" is sufficient instead of "homerce book"?

Reply:
Okay, will rename. Apparently there are many instances of "Homerce book" throughout the project. I think a "find-and-replace" went wrong in the past.

Reply:
Yes I think it happened during the refactoring from Addressbook to Homerce